### DISCUSSION TITLE
How to find out the polling frequency in version 1.0.2?

### DISCUSSION THREAD

**Discussion by aLeeTheY** [mat1jaczyyy/Keyboard-Inspector#42] [Help]
Hello. How can I find out the polling frequency of my keyboard using your software? In the early versions of your program, everything was very clear, but in the later versions you introduced graphs, according to which I do not understand what the polling frequency is on my device. Can you improve the interface, add for example a field where a figure would be displayed, at least an approximate or average, of the polling frequency of the USB device???

**Comment by Zepheniahh** [comment]
As development of Keyboard Inspector continued, a lot was learnt about the behaviour of not only keyboards but USB devices in general. It became evident that getting the program to spit out a number as the performance based on some heuristic was impossible since device behaviour is much more complex. Therefore Keyboard Inspector elects to instead present a few different analyses to the user, letting the user understand their device far better than any heuristic ever could. This has downsides, foremost that it raises the bar of entry to using Keyboard Inspector.
That said, a more detailed guide than that currently on github is being written.
